app/main.py

```diff
@@ -1,2 +1,118 @@
 class Dictionary:
-    pass
+    LOAD_FACTOR = 2 / 3
+    RESIZE = 2
+    DEFAULT_CAPACITY = 8
+
+    def __init__(self):
+        self.initial_capacity = 8
+        self.storage = [[] for _ in range(self.DEFAULT_CAPACITY)]
+        self.length = 0
+
+    def resize(self):
+        saved_storage = self.storage[:]
+        self.length = 0
+        self.initial_capacity *= self.RESIZE
```

Comment:
If you change the value of the initial capacity maybe change its name to the just `capacity`?